openssl.BUILD

```diff
@@ -127,6 +105,24 @@ genrule(
         "install/include/openssl/asn1t.h",
         "install/include/openssl/uierr.h",
         "install/include/openssl/rsaerr.h",
+]
+
+cc_library(
+    name = "ssl",
+    srcs = [
+        "install/lib/libssl.a",
+        "install/lib/libcrypto.a",
+    ] + openssl_headers,
+    hdrs = glob(["install/include/openssl/*.h"]),
+    includes = ["install/include"],
+    visibility = ["//visibility:public"],
+)
+
+genrule(
```

Comment:
It might be that we should investigate using `foreign_cc` (https://github.com/bazelbuild/rules_foreign_cc) here for building OpenSSL because Envoy is using that for many (all?) non-bazel external deps.

Reply:
There's a doc that talks about the choices: https://github.com/envoyproxy/envoy/blob/master/bazel/EXTERNAL_DEPS.md

that suggests `genrule` for this purpose (and is what, e.g., boringssl-fips uses too).

Reply:
I think `foreign_cc` support was added to Envoy after that file was last updated, but I'm fine with `genrule` too.